src/core/tracking/google/GoogleAnalytics.js

```diff
@@ -4,6 +4,6 @@
 
 const GOOGLE_TRACKING_ID :string = 'UA-118446829-4';
 
-export {
+export default {
```

Comment:
Not necessary.

Reply:
linter says 'export default' is preferred.

Reply:
It's a warning, not an error.